### DISCUSSION TITLE
Welcome to Moq Discussions!

### DISCUSSION THREAD

**Discussion by kzu** [devlooped/moq#1413] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by belgacem17** [comment]
I added dll  reference's of another project, I used xunit, I want to moq the functions of it, like this code 
```
public bool Test(string value)
 {
   // using ACPUtilityManager.HelperUtility: ACPUtilityManager is reference of another project librairy  
   // WebServiceUtility: static class , testMoq(value)  method of this class  
     var ttt = WebServiceUtility.testMoq(value);
    
     return true;
 }
```
but i can't moq this a class, help me please
